packages/x-date-pickers-pro/src/internal/hooks/useRangePosition.ts

```diff
@@ -0,0 +1,42 @@
+import useControlled from '@mui/utils/useControlled';
+import useEventCallback from '@mui/utils/useEventCallback';
+import { RangePosition } from '../models/range';
+
+export interface UseRangePositionProps {
+  /**
+   * The position in the range being edited.
+   * Used when the component position is controlled.
+   */
+  rangePosition?: RangePosition;
+  /**
+   * The default position in the range being editing value.
```

Comment:
Is it just me or is this line somewhat clumsy and unclear? 🤔 
Maybe just this?
```suggestion
   * The default position in the range being edited.
```
Or (because "being edited" part might somewhat imply that you need to start editing the range for this prop to take effect, even though, at least for static pickers it's clearly evident that it changes the focused toolbar range position):
```suggestion
   * The default position in the range.
```

Reply:
I think what I meant is `the position currently being edited in the range` for `rangePosition` and `the position edited on mount in the range`.

I think we need to keep the `edited` or `value` information, otherwise `position in the range` alone can be hard to understand.

Reply:
> I think we need to keep the `edited` or `value` information

That is totally fine. 👌 It would be more in line with other comments on similar props.
I'd be hesitant on using the second suggestion, just because we do not have cases of it yet.
Maybe it wouldn't hurt letting users learn how do these `default` prop actually work/behave instead of specifying `on mount.` Especially when we have cases like `defaultValue` and lot's of demos using it, hence, IMHO, `default` shouldn't be an issue for users to understand. 🤞 🤔